### PR TITLE
[AzureStack]PowerShell update for Fabriclocation Module

### PR DESCRIPTION
**Pull request by LingyunSu**
## Checklist

- [x] I have read the [_Submitting Changes_](../blob/master/CONTRIBUTING.md#submitting-changes) section of [`CONTRIBUTING.md`](../blob/master/CONTRIBUTING.md)
- [x] The title of the PR is clear and informative
- [x] The appropriate `ChangeLog.md` file(s) has been updated:
    - For any service, the `ChangeLog.md` file can be found at `src/{{SERVICE}}/{{SERVICE}}/ChangeLog.md`
    - A snippet outlining the change(s) made in the PR should be written under the `## Upcoming Release` header -- no new version header should be added
- [x] The PR does not introduce [breaking changes](../blob/master/documentation/breaking-changes/breaking-changes-definition.md)
- [x] If applicable, the changes made in the PR have proper test coverage
- [x] For public API changes to cmdlets:
    - [ ] a cmdlet design review was approved for the changes in [this repository](https://github.com/Azure/azure-powershell-cmdlet-review-pr) (_Microsoft internal only_)
    - [ ] the markdown help files have been regenerated using the commands listed [here](../blob/master/documentation/development-docs/help-generation.md#updating-all-markdown-files-in-a-module)

### CODE REVIEW

**Comment by adxsdkps** [comment]
Can one of the admins verify this patch?

**Comment by LingyunSu** [comment]
@bganapa @JesusAlvarezTorres @vikasnav

**Comment by LingyunSu** [comment]
@bganapa SDK was build, and this new update of version for other files

File: src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1 [hunk 1]
```diff
@@ -17,7 +17,7 @@ Licensed under the MIT License. See License.txt in the project root for license
     RootModule        = 'Azs.Fabric.Admin.psm1'
 
     # Version number of this module.
-    ModuleVersion     = '0.5.0'
+    ModuleVersion     = '0.5.1'
```
Comment (bganapa) [inline src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1]: no ned to update the version, pls revert

File: src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1 [hunk 2]
```diff
@@ -129,10 +129,8 @@ Licensed under the MIT License. See License.txt in the project root for license
 
             # ReleaseNotes of this module
             ReleaseNotes = '
-            * Breaking changes for Drive and Volume resources with API version 2019-05-01: (The features are supported by Azure Stack 1910+)
```
Comment (bganapa) [inline src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1]: please keep this release notes

File: src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1 [hunk 3]
```diff
@@ -133,6 +133,8 @@ Licensed under the MIT License. See License.txt in the project root for license
                 * The value of Id, Name, HealthStatus and OperationalStatus have been changed.
                 * We have supported new properties FirmwareVersion, IsIndicationEnabled, Manufacturer and StoragePool for Drive resources.
                 * The properties CanPool and CannotPoolReason of Drive resources have been deprecated, please use OperationalStatus instead.
+            * Module updated for Fabriclocation
```
Comment (bganapa) [inline src/StackAdmin/Azs.Fabric.Admin/Module/Azs.Fabric.Admin/Azs.Fabric.Admin.psd1]: I think you meant Model here instead of Module

**Comment by wyunchi-ms** [comment]
BTW, could you please rebase these commits into one? Because these are small changes.

**Comment by LingyunSu** [comment]
@wyunchi-ms Thanks, rebase done, all squashed into one.

**Comment by wyunchi-ms** [comment]
Hi @LingyunSu could you please take a look at the CI? Do you want to merge the code into `Azure:AzureRM`? Is this you really want?

**Comment by wyunchi-ms** [comment]
The jenkins machine is unavailable now. I will retry the CI after the machine is back.

**Comment by markcowl** [comment]
@azuresdkci test this please

**Pull request merged**